### ISSUE TITLE
Any file open dialog crashes on Arch Linux

### ISSUE THREAD

**Issue by Jesuszilla** [microsoft/vscode#107392]
- VSCode Version: 1.49.1
- OS Version: Linux x64 5.8.10-arch1-1
Commit: 58bb7b2331731bf72587010e943852e13e6fd3cf
Date: 2020-09-23T22:26:38.399Z
Electron: 9.2.1
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0

KDE Plasma is my desktop environment.

Steps to Reproduce:

1. Start VS Code
2. Try to open a folder, file, or workspace.
3. VS Code crashes without any error.

Does this issue occur when all extensions are disabled?: Yes

**Comment by deepak1556** [comment]
Did you install vscode via snap ?

**Comment by Jesuszilla** [comment]
> Did you install vscode via snap ?

No, I did not. I used pacman.

**Comment by deepak1556** [comment]
Update to latest stable `1.49.2` and launch with `code --crash-reporter-directory /tmp/vscode-crash`, once the crash happens by opening any dialog, attach the generated crash dumps from that directory. Thanks!

**Comment by Jesuszilla** [comment]
> Update to latest stable `1.49.2` and launch with `code --crash-reporter-directory /tmp/vscode-crash`, once the crash happens by opening any dialog, attach the generated crash dumps from that directory. Thanks!


[vscode-crash.tar.gz](https://github.com/microsoft/vscode/files/5301636/vscode-crash.tar.gz)

**Comment by deepak1556** [comment]
Thanks for the dump, currently there are missing symbols for 

```
libgio-2.0.so.0  ???  (WARNING: No symbols, libgio-2.0.so.0, A7EBE401A46DA7B4E61D629A75BDD33A0)
 libglib-2.0.so.0  ???  (WARNING: No symbols, libglib-2.0.so.0, 39FBCD6566D23502051F5CEF4A9171CF0)
libgobject-2.0.so.0  ???  (WARNING: No symbols, libgobject-2.0.so.0, FDE33F09D68AFE6DEC83B393C87B153E0)
```

we are using https://symbols.mozilla.org/ as the symbol server. Do you know where I can get debug versions of these archlinux packages ?

**Comment by Jesuszilla** [comment]
> Thanks for the dump, currently there are missing symbols for the following in the server we depend on https://symbols.mozilla.org/
> 
> ```
> libgio-2.0.so.0  ???  (WARNING: No symbols, libgio-2.0.so.0, A7EBE401A46DA7B4E61D629A75BDD33A0)
>  libglib-2.0.so.0  ???  (WARNING: No symbols, libglib-2.0.so.0, 39FBCD6566D23502051F5CEF4A9171CF0)
> libgobject-2.0.so.0  ???  (WARNING: No symbols, libgobject-2.0.so.0, FDE33F09D68AFE6DEC83B393C87B153E0)
> ```
> 
> Do you know where I can get debug versions of these archlinux packages ?

I'm not 100% sure, but my bet is on the `eschwartz` repository, according to https://wiki.archlinux.org/index.php/Debug_-_Getting_Traces#GTK3/GLib2

**Comment by deepak1556** [comment]
yup that repo contained the debug versions, I will try to symbolicate them tomorrow. Side tracked on another issue today.

**Comment by Jesuszilla** [comment]
> yup that repo contained the debug versions, I will try to symbolicate them tomorrow. Side tracked on another issue today.

Hello! Any update on this?

**Comment by maxekman** [comment]
I think I might have this issue. In my case running Manjaro with VSCode from Snap.

**Comment by charliebarber** [comment]
This issue also happens for me on Manjaro when trying to close an opened and unsaved file or delete a file in the file explorer.

**Comment by deepak1556** [comment]
Can everyone check if the latest insiders fixes the issue https://code.visualstudio.com/insiders, thanks!

**Comment by charliebarber** [comment]
> Can everyone check if the latest insiders fixes the issue https://code.visualstudio.com/insiders, thanks!

Fixed it for me

**Comment by DB-CL** [comment]
> Fixed it for me

Oh shoot, exact same behavior for me with `code-insiders_1.51.0-1604433932_amd64.deb`

**Comment by maxekman** [comment]
This is fixed for me in 1.51.0 (non-insiders).

**Comment by DB-CL** [comment]
I've tried 1.51.0, still the same :(
